src/server.jl

```diff
@@ -333,8 +341,8 @@ function raw_markdown_chunks_from_string(path::String, markdown::String)
             # all other options seem to be quarto-rendering related, like where to put figure captions etc.
             source = node.literal
             cell_options = extract_cell_options(source; file = path, line = line)
-            evaluate = get(cell_options, "eval", true)
-            if !(evaluate isa Bool)
+            evaluate = get(cell_options, "eval", Unset())
```

Comment:
I could have tried to route the options through the functions that extract the cell options but that would have been a larger change because those are currently independent. And I didn't take the usual `nothing` because that's a value that you can also set in YAML, and I wanted to error for invalid values instead I think, so I chose to make a new value that's outside of the YAML set.

Reply:
> that would have been a larger change because those are currently independent

How much larger? Larger changes aren't always the wrong thing to do. If logically these options aren't independent then adjusting the code so that there is dependence between them makes sense to me.

Reply:
Cell parsing can be independent from global settings. If you do the extra work to route the global options into parsing that saves you from having to use a placeholder value for "nothing was specified here". A cell with `eval: true` should be evaluated in a `eval: false` global regime, but the default is also implicit `eval: true`. So one needs to disambiguate implicit and explicit true. I do that with this placeholder. We could also decide that the implicit `true` is actually `eval: null` or whatever that is in YAML. Right now, I think it would error if the user wrote that. But we could make it so that just means whatever the global default is. Then we don't need `Unset`.

Reply:
Just stick with what's here then for now.